bigframes/core/indexers.py

```diff
@@ -215,7 +215,7 @@ def __getitem__(self, key: tuple) -> bigframes.core.scalar.Scalar:
             raise ValueError(error_message)
         if len(key) != 2:
             raise TypeError(error_message)
-        block: bigframes.core.blocks.Block = self._dataframe._block
+        block: bigframes.core.blocks.Block = self._dataframe._block  # type: ignore
```

Comment:
Not sure why I needed this to make pre-commit happy as `nox -r -s mypy` succeeds without this.